js-miniapp-bridge/src/types/token-data.ts

```diff
@@ -2,15 +2,35 @@
 export interface NativeTokenData {
   token: string;
   validUntil: number;
+  scopes?: NativeTokenPermission;
```

Comment:
I think this isn't the correct return format, is it? We changed the iOS/Android SDK so that it returns `audience` and `scope` properties separately. i.e:
```
export interface NativeTokenData {
  token: string;
  validUntil: number;
  audience: string;
  scopes: string[];
}
```

Reply:
I'm just confused after discussing with @minh-rakuten, I just have to re-focus on this point. For now it's working with the current PR on iOS side, I really have to check with @minh-rakuten as it's the format I understood he also expected 😭

Reply:
Oh sorry I think I caused some confusion.

When I commented on the Android PR, I meant that it should change to match iOS with having audience and scope not be under a separate object.

But if what you have here is already working on both platforms, then I don't think we need to change it. Just probably the naming should be different:
- `NativeTokenScopes`
- `AccessTokenScopes`

Reply:
done